core/trino-main/src/main/java/io/trino/execution/executor/TaskExecutor.java

```diff
@@ -164,20 +176,25 @@ public TaskExecutor(TaskManagerConfig config, VersionEmbedder versionEmbedder, M
                 config.getMinDriversPerTask(),
                 config.getMaxDriversPerTask(),
                 versionEmbedder,
+                config.getInterruptRunawaySplitsTimeout(),
+                DEFAULT_INTERRUPTIBLE_SPLIT_PREDICATE,
+                DEFAULT_INTERRUPT_SPLIT_INTERVAL,
                 splitQueue,
                 Ticker.systemTicker());
     }
 
     @VisibleForTesting
     public TaskExecutor(int runnerThreads, int minDrivers, int guaranteedNumberOfDriversPerTask, int maximumNumberOfDriversPerTask, Ticker ticker)
     {
-        this(runnerThreads, minDrivers, guaranteedNumberOfDriversPerTask, maximumNumberOfDriversPerTask, testingVersionEmbedder(), new MultilevelSplitQueue(2), ticker);
+        this(runnerThreads, minDrivers, guaranteedNumberOfDriversPerTask, maximumNumberOfDriversPerTask, testingVersionEmbedder(),
```

Comment:
nit: put everything on new line for consistency with other places

Reply:
done